doc/src/manual/faq.md

```diff
@@ -834,6 +834,22 @@ a scalar can participate in linear algebra operations such as `2 * rand(2,2)`,
 but the analogous operation with a zero-dimensional array
 `fill(2) * rand(2,2)` is an error.
 
+## Computing cluster
+
+### How do I manage precompilation cache files in distributed file systems?
+
+When using `julia` in high-performance computing (HPC) facilities, invoking
+_n_ `julia` processes simultaneously creates at most _n_ temporary copies of
+cache files.  It may become a major issue in slow and/or small distributed
+file systems.  There are a few possible workarounds:
+
+1. Use `julia` with `--compilecache=no` flag to turn off precompilation.
```

Comment:
This option has been renamed: https://github.com/JuliaLang/julia/pull/23616/files

compilecache -> compiled-modules